network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/figure_grid_block.html

```diff
@@ -0,0 +1,9 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="figure-grid" style="display: flex; flex-direction: row; ">
```

Comment:
this is pretty terrible, but in order to "see it work" I added a hardcoded style here, rather than add in a rule for `.figure-grid` and descendants to our CSS file and probably getting the "where to put that" wrong.